R/page_size.R

```diff
@@ -3,8 +3,63 @@
 NULL
 ## https://www.ietf.org/rfc/rfc0678.txt
 
-## This assumes fixed font size, monospaced font
+times_font_name <- function() {
+  plat <- .Platform$OS.type
+  switch(plat,
+    "unix" = "Times",
```

Comment:
Is mac also here? I think it should be unix-like right?

Reply:
Yes, mac is also a "unix" os type